docs/index.rst

```diff
@@ -150,6 +150,19 @@ extension types <using-extension-types>` documentation pages.
 
     data-types
 
+Support Utilities
+=================
+
+The package bundles a few support and utility functions that try to fill a few
+gaps you will observe when working with CrateDB, when compared with other
+databases. As a distributed OLAP database, it naturally does not include certain
```

Comment:
I think that we shouldn't mention as a `distributed OLAP`, specially combining it with `naturally` after that, I would simplify and say that `CrateDB's behavior and features deviate from those found usually in an RDBMS`.

Reply:
Maybe something like "Due to its distributed nature, CrateDB's behavior and features differ from those found in other RDBMS systems".